playwright-tests/tests/payments/vote-on-request.spec.js

```diff
@@ -42,14 +42,21 @@ async function voteOnProposal({
       return originalResult;
     },
   });
+
   await mockRpcRequest({
     page,
     filterParams: {
       method_name: "get_proposal",
     },
     modifyOriginalResultFunction: (originalResult) => {
       originalResult = TransferProposalData;
-      if (isTransactionCompleted) {
+      if (isTransactionCompleted && vote === "Remove" && !isMultiVote) {
+        return {
+          isError: true,
+          error:
+            "wasm execution failed with error: HostError(GuestPanic { panic_msg: \"panicked at 'ERR_NO_PROPOSAL', sputnikdao2/src/views.rs:102:48\" })",
```

Comment:
It would be better to provoke this scenario in the sandbox, rather than mocking it here. We are not sure if later versions of the sputnikdao will fail on line 102 either.

Reply:
yeah I agree 100%, I created a [ticket](https://github.com/NEAR-DevHub/neardevhub-treasury-dashboard/issues/263) for it